draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -753,6 +753,80 @@ any stream which references the failing certificate in question or process the
 requests as unauthenticated and provide error information at the HTTP semantic
 layer.
 
+# Certificate Extensions {#extension}
+
+## Allowed Primary Domain Certificate Extension {#extension-primary}
+
+The Allowed Primary Domain extension allows certificates to limit their use with
+Secondary Certificate Authentication.  The identities listed in this extension
+are restrictions asserted by the requester of the certificate and are not
+verified by the CA.  Conforming CAs SHOULD mark the allowedPrimaryDomain
+extension as non-critical.
+
+Each allowed primary domain is represented as an IA5String. The name MUST be in
+the "preferred name syntax", as specified by Section 3.5 of {{!RFC1034}} and as
+modified by Section 2.1 of {{!RFC1123}}.  Note that while uppercase and
+lowercase letters are allowed in domain names, no significance is attached to
+the case. In addition, while the string " " is a legal domain name,
+affiliatedDomain extensions MUST NOT contain " ".  Rules for encoding
+internationalized domain names are specified in Section 7.2 of {{!RFC5280}}.
```

Comment:
Does this crib from text elsewhere?  Can that text be included by reference instead?  The reason I ask is that this entire section seems to be copied from something, and I don't know if it is the same as that other thing, or subtly different.

Reply:
Cribs a lot from the definition of SAN, but cuts out the allowance for different name types since this is specifically a domain name.

Reply:
I think that you could just copy the SAN grammar and say that the rules are inherited.  I know that means that you get other name types, but I don't think that's fatal (we still accept IP SANs, and probably wouldn't have a problem if they were included in this).

Reply:
I'm okay either way, but I think I'd like broader WG opinion on whether we want the extra feature of non-domain requirements.